tripy/tests/frontend/test_cache.py

```diff
@@ -0,0 +1,103 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pytest
+import os
+
+import tripy as tp
+
+from tripy.frontend.trace import Trace
+from tripy.frontend.cache import ExecutableCache
+
+
+@pytest.fixture
+def set_env(monkeypatch):
+    monkeypatch.setenv("TRIPY_EAGER_CACHE", "1")
```

Comment:
There's a `helper.config` context manager that can temporarily change the config for the current test

Reply:
Sounds good! will use it